tests/test_fields.py

```diff
@@ -17,6 +18,12 @@
 )
 
 
+def field_names(model):
+    """"Return field names for `model`."""
+    fields = model._meta.get_fields()
+    return {field.name for field in fields}
```

Comment:
[`find_packages`](https://github.com/incuna/django-pgcrypto-fields/blob/master/setup.py#L9) will pick up on the `tests` folder, and package it as a python module that gets installed by pip. Probably best to either exclude `tests` from the packages, or just replace `find_packages` with `['pgcrypto']` (or `['incuna_test_utils']` in the case of the test package).